data/2mass_j10585054-2346206.json

```diff
@@ -146,5 +146,14 @@
             "comments": null,
             "reference": "Gira11"
         }
+    ],
+    "RadialVelocities": [
+        {
+            "radial_velocity": 11.02063274383545,
+            "radial_velocity_error": 37.235923767089844,
```

Comment:
Do we want to ingest RVs that have a very large uncertainty?  @Will-Cooper , thoughts?

Reply:
That's a bigger discussion @kelle; I wouldn't trust any RVs for UCDs from Gaia to do any science but is the idea of the database to record all forms of data or just what we are saying is trustworthy? I see the Gaia G mag is 13.7 and the limit for RV processing is 14, so it's borderline snuck in there at the most uncertain bound.